### PR TITLE
Remove magic number from handshake

### PR DESCRIPTION
**Pull request by GalRogozinski**
# Description

While looking over networking code I found a magic number that confused me

## Type of change

- Readability fix 

# How Has This Been Tested?

If all unit tests and regression test pass it should suffice

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by GalRogozinski** [comment]
Quick explanation:
`(maxLength - (maxLength - 60) + Protocol.SUPPORTED_PROTOCOL_VERSIONS.length` =  `60 + Protocol.SUPPORTED_PROTOCOL_VERSIONS.length`

It was unclear where 60 comes from, so it was replaced constants

File: src/main/java/com/iota/iri/network/protocol/Handshake.java [hunk 1]
```diff
@@ -36,15 +43,14 @@
...
-        ByteBuffer buf = ByteBuffer.allocate(ProtocolMessage.HEADER.getMaxLength() + payloadLengthBytes);
-        Protocol.addProtocolHeader(buf, ProtocolMessage.HANDSHAKE, payloadLengthBytes);
+        ByteBuffer buf = ByteBuffer.allocate(ProtocolMessage.HEADER.getMaxLength() + PAYLOAD_LENGTH_BYTES);
+        Protocol.addProtocolHeader(buf, ProtocolMessage.HANDSHAKE, PAYLOAD_LENGTH_BYTES);
         buf.putChar(ownSourcePort);
         buf.putLong(System.currentTimeMillis());
         buf.put(ownByteEncodedCooAddress);
         buf.put(ownUsedMWM);
         buf.put(Protocol.SUPPORTED_PROTOCOL_VERSIONS);
+        //update PAYLOAD_LENGTH_BYTES if changed
```
Comment (acha-bill) [inline src/main/java/com/iota/iri/network/protocol/Handshake.java]: Is this comment necessary?

Comment (GalRogozinski) [reply src/main/java/com/iota/iri/network/protocol/Handshake.java]: So the programmer won't forget to update this value if in the future another `buf.put` is added
you think it is too obvious?

Comment (acha-bill) [reply src/main/java/com/iota/iri/network/protocol/Handshake.java]: It's not "too" obvious. It's fine.

**Pull request merged**